src/Containers/HorizontalScrollList.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { flex, scroll } from '../Utils/Mixins';
+import { Select } from '../Inputs/Select';
+
+export interface ScrollListProps extends React.HTMLAttributes<HTMLDivElement> {
+    labelArray: string[];
+    menuName: string;
+    menuWidth: number;
+    labelPadding: string;
+    hoveredStyle?: Function;
+    selectedStyle?: Function;
+}
+
+export interface ScrollListDivProps {
+    menuWidth?: number;
+}
+
+export interface ListItemProps {
+    label: string;
+    hoveredStyle: Function;
+    selectedStyle: Function;
```

Comment:
Unsure if styling should be passed as a function that returns a string, as a standalone string, or as some other data type. Looked into React.CSSProperties but was unable to figure anything out